src/js/controllers/walletHome.js

```diff
@@ -28,6 +28,27 @@ angular.module('copayApp.controllers').controller('walletHomeController', functi
   this.addr = {};
   this.lockedCurrentFeePerKb = null;
 
+  self.hideBalance = function() {
+    var fc = profileService.focusedClient;
+    storageService.getHideBalanceFlag(fc.credentials.walletId, function(err, shouldHideBalance) {
+      if (err) self.shouldHideBalance = false;
+      else self.shouldHideBalance = shouldHideBalance ? true : false;
+    });
+  }
+  self.hideBalance();
+
+  self.onMouseDown = function() {
+    var fc = profileService.focusedClient;
+    self.mouseDown = $timeout(function() {
+      self.shouldHideBalance = self.shouldHideBalance ? false : true;
```

Comment:
Self.shouldHideBalance should change from false to true when the balance is pressed for half a second, and also from true to false when the lock image is pressed.